sagemaker-experiments/mnist-handwritten-digits-classification-experiment.ipynb

```diff
@@ -449,20 +437,103 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {
-    "scrolled": true
-   },
+   "metadata": {},
    "outputs": [],
    "source": [
     "lineage_table.dataframe()"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Deploy endpoint for the best training-job / trial component\n",
```

Comment:
seems unrelated to experiments. link to a deploy related notebook or documentation.

Reply:
Fair comment; wiki doc example that has some issues shows going from a TC to deployed model.  I opened issues with the relevant parties to correct the Studio experience and the doc for the example.  In the mean time this keeps that same example flow which seemed reasonable.  (Should be stable over time as well.)